NDVI_seminat.Rmd

```diff
@@ -0,0 +1,989 @@
+summary(model.seminat.time.tilst.T31.Sent)$coefficients$cond
+summary(model.seminat.time.tilst.T31.Modi)$coefficients$cond
+summary(model.seminat.time.tilst.T31.Land)$coefficients$cond
+```
+The temporal NDVI increase in boreal heath has NOT been stronger with reduced ecological condition. So a temporal NDVI approach does not seem like a promising way to go.
+We check for the same pattern in T32 and T34, jsut because they are the other semi-natural major ecosystem types with a substantial number of occurrences in the the NiN data:
+```{r, warning=FALSE}
+model.seminat.time.tilst.T32.Sent <- glmmTMB(mean_beta ~ year * tilstand + (1 | id), family = beta_family(), data = SentinelNDVI.seminat[SentinelNDVI.seminat$hovedtype %in% c("T32"), ])
+model.seminat.time.tilst.T32.Modi <- glmmTMB(mean_beta ~ year * tilstand + (1 | id), family = beta_family(), data = ModisNDVI.seminat[ModisNDVI.seminat$hovedtype %in% c("T32"), ])
+model.seminat.time.tilst.T32.Land <- glmmTMB(mean_beta ~ year * tilstand + (1 | id), family = beta_family(), data = LandsatNDVI.seminat[LandsatNDVI.seminat$hovedtype %in% c("T32"), ])
+summary(model.seminat.time.tilst.T32.Sent)$coefficients$cond
+summary(model.seminat.time.tilst.T32.Modi)$coefficients$cond
+summary(model.seminat.time.tilst.T32.Land)$coefficients$cond
+
+model.seminat.time.tilst.T34.Sent <- glmmTMB(mean_beta ~ year * tilstand + (1 | id), family = beta_family(), data = SentinelNDVI.seminat[SentinelNDVI.seminat$hovedtype %in% c("T34"), ])
+model.seminat.time.tilst.T34.Modi <- glmmTMB(mean_beta ~ year * tilstand + (1 | id), family = beta_family(), data = ModisNDVI.seminat[ModisNDVI.seminat$hovedtype %in% c("T34"), ])
+model.seminat.time.tilst.T34.Land <- glmmTMB(mean_beta ~ year * tilstand + (1 | id), family = beta_family(), data = LandsatNDVI.seminat[LandsatNDVI.seminat$hovedtype %in% c("T34"), ])
+summary(model.seminat.time.tilst.T34.Sent)$coefficients$cond
+summary(model.seminat.time.tilst.T34.Modi)$coefficients$cond
+summary(model.seminat.time.tilst.T34.Land)$coefficients$cond
+```
+Again, patterns are only identifiable with Sentinel and Landsat data, but not with Modis. Semi-natural grasslands (T32) do indeed show a stronger increase in NDVI as condition deteriorates, so here a temporal approach may be useful under a reduced land-management/shrub-/tree-encroachment scenario. Coastal heathlands (T34), on the other hand, do not show stronger temporal increases with reduced condition. Coastal heathlands are subject to a management cycle, which includes developmental and successional changes over a course of 15-25 years followed by prescribed burning which resets that process. So, we have to expect increases in greenness in coastal heathlands over time also in well-managed areas, and thus temporal changes in NDVI are largely irrelevant for assessing ecological condition.
+
+# Overall conclusion
```

Comment:
Note that single # causes problems in bookdown. I remove them.

Reply:
ok, thanks